frontend/packages/ceph-storage-plugin/src/components/ocs-install/node-list.tsx

```diff
@@ -343,12 +343,16 @@ const CustomNodeTable: React.FC<CustomNodeTableProps> = ({
       .then((storageClasses: StorageClassResourceKind[]) => {
         // find all storageclass with the given provisioner
         const scList = _.filter(storageClasses, (sc) => sc.provisioner === provisioner);
-        // take the default storageclass
+        // take the provisioner based storageclass
         _.forEach(scList, (sc) => {
           if (isDefaultClass(sc)) {
             storageClass = sc.metadata.name;
           }
         });
+        // take the first storageclass if default not set
+        if (!storageClass && storageClass.length > 0) {
+          storageClass = getName(_.get(scList, '0'));
+        }
```

Comment:
@cloudbehl Also we need to handle the scenario, when no storage class is present. UI should throw error.

Reply:
@gnehapk I want the backend to throw an error on this one.

Reply:
I don't want to write logic to throw error messages from UI. What you think?

Reply:
Agree.